config/config.go

```diff
@@ -48,20 +48,29 @@ func (g Generator) ParamString() string {
 	return strings.Join(params, ",")
 }
 
-func (g Generator) ParamStringWithOut() string {
-	outPath := g.OutPath()
+// ParamStringWithOut will return the generator paramaters formatted
+// for protoc, including where protoc should output the generated files.
+// It will use 'gunkPackage' if no 'out' key was set in the config.
+func (g Generator) ParamStringWithOut(gunkPackage string) string {
```

Comment:
assuming the parameter here is the package's directory, I'd call this `packageDir`

Reply:
Done.